frontend/src/App.js

```diff
@@ -200,6 +201,15 @@ function App() {
             <AppWrapper authenticated logout={logout}><RegionalDashboard user={user} /></AppWrapper>
           )}
         />
+        <Route
```

Comment:
Does this need to be wrapped in a feature flag?

Reply:
yeah, good catch. done here: [1651670](https://github.com/HHS/Head-Start-TTADP/pull/1373/commits/16516704d15dee0db6f9d0bc04c782a7f44281bd)